dll/win32/shdocvw/mrulist.cpp

```diff
@@ -32,11 +33,13 @@ class CMruBase;
             class CMruPidlList;
 class CMruClassFactory;
 
-extern "C" void __cxa_pure_virtual(void)
+#ifdef __GNUC__ // GCC and Clang.
+extern "C" void __cxa_pure_virtual()
```

Comment:
Since then, this very occurrence was moved on 960a305, then removed/replaced on #7467 678aa63.